tests/functional/export_v2.t

```diff
@@ -65,3 +65,25 @@ Export with auspice config JSON with an extensions block
   $ python3 "$TESTDIR/../../scripts/diff_jsons.py"  export_v2/dataset2.json "$TMP/dataset3.json" \
   >   --exclude-paths "root['meta']['updated']"
   {}
+
+# export_v2/location_node-data2.json was produced by a mismatched version
+# Running without --skip-validation should result in an error
+# Message printed: "Validation of auspice_config3.json failed."
+  $ ${AUGUR} export v2 \
+  >   --tree export_v2/tree.nwk \
+  >   --node-data export_v2/div_node-data.json export_v2/location_node-data2.json \
+  >   --auspice-config export_v2/auspice_config3.json \
+  >   --output "$TMP/dataset4.json" &>/dev/null
+  [1]
```

Comment:
Instead of commenting what the message should be and piping stderr to null, I would allow stderr to print and put the expected message in the output of the command above `[1]`. This way the Cram test documents and tests the expected behavior.

Reply:
Addressed in [404595d](https://github.com/nextstrain/augur/pull/902/commits/404595d60759a46f3294154ec37211d5ea907feb). A bit tricky as we print the current augur version so need to use a regex on the output.